apps/api/src/campaign/campaign.controller.ts

```diff
@@ -28,7 +28,6 @@ export class CampaignController {
   ) {}
 
   @Get('list')
-  @Public()
```

Comment:
We need to leave this one public as it used on the main campaigns page.

Reply:
In the main campaign page should be returned only active and completed campaigns. I thought this route to be used for admin list with campaigns and new one to be created for the main page.

Reply:
Actually in the existing one should be added a filter for active and completed and new route with admin access only could be added.